packages/wonder-blocks-tooltip/components/tooltip-portal-mounter.js

```diff
@@ -111,24 +99,34 @@ export default class TooltipPortalMounter extends React.Component<Props> {
         destination.parentNode.removeChild(destination);
     }
 
-    _timeoutDoMount() {
-        this._clearMountTimeout();
-        this._timeoutId = setTimeout(() => this._doMount(), 0);
-    }
+    _renderChildren() {
+        if (!this._destination) {
+            this._doMount();
+            return;
+        }
+
+        const {children} = this.props;
+        if (!children) {
+            this._doUnmount();
+            return;
+        } else {
+            ReactDOM.unmountComponentAtNode(this._destination);
+        }
 
-    _timeoutDoUnmount() {
-        this._clearMountTimeout();
-        this._timeoutId = setTimeout(() => this._doUnmount(), 0);
+        // We have to render the subtree like this so that everything works as
+        // expected.
+        // See https://github.com/tajo/react-portal/blob/master/src/LegacyPortal.js
+        ReactDOM.unstable_renderSubtreeIntoContainer(
```

Comment:
Will this still be necessary if/when we switch to `ReactDOM.createPortal(child, container)`?

Reply:
No, I don't think so. Assuming that the react portal works how we need it to, we shouldn't need to do this magic.